Source/minitext.cpp

```diff
@@ -86,8 +86,8 @@ void InitQTextMsg(int m)
 
 void DrawQTextBack()
 {
-	CelDraw(PANEL_X + 24, 487, pTextBoxCels, 1, 591);
-	trans_rect(PANEL_LEFT + 27, 28, 585, 297);
+	CelDraw(PANEL_X + 24, (SCREEN_Y + 327 + UI_OFFSET_Y), pTextBoxCels, 1, 591);  //487  0.683

```

Comment:
What is `0.683` a reference to?

Reply:
It was probably some math I was doing when trying to see how things worked. I definitely should have deleted it earlier. My bad.